azkaban-web-server/src/test/resources/create.quartz-all-tables.sql

```diff
@@ -0,0 +1,165 @@
+-- All necessary quartz table create statement collection for unit test purpose
```

Comment:
If the sql statements are changed incorrectly and this file is not updated, the tests may pass.

Can you think of ways to remove this duplication?

Reply:
I can not think of situations where we will change quartz SQL statements. Do I miss something?

Reply:
When we update Quartz, the sql statements may change. Am I right?

Reply:
Upgrade Quartz! Yes, that's quite possible.